src/utils.ts

```diff
@@ -203,3 +203,12 @@ const urlsWebApp: any = [
 export function getCurrentPageForUrl(currentUrl: string): any {
   return urlsWebApp.find((item: any) => item.url === currentUrl);
 }
+
+export function findUrlInRetryList(currentUrl: any): any {
+  const urls = [
+    'WebAppPublic/Login',
+    'WebAppPublic/CreateUser',
+    '/WebAppPublic/SendResetPasswordEmail',
```

Comment:
Why this one has a different format?

Reply:
because in doppler legacy has the `/` maybe need delete both sides

Reply:
And it is different in the other cases?

Reply:
yes other cases has the `/` too. maybe after can add refactor to all urls